File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int sum = (n*(n+1))/2;
        int nsum = 0;
        for(int i = 0; i<n; i++){
            nsum += nums[i];
        }
        return sum-nsum;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int tsum = (n * (n + 1)) / 2;
        int nsum = nums[0];
        for (int i = 1; i < nums.length; i++) {
            nsum += nums[i];
        }
        return tsum - nsum;
    }
}